src/utils/users.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import Client from 'client';
+import {UsersTypes} from 'constants';
+const HTTP_UNAUTHORIZED = 401;
+
+export function profilesToSet(state, action) {
+    const nextState = {...state};
+    if (action.offset != null && action.count != null) {
+        nextState.offset = action.offset + action.count;
+        nextState.count += action.count;
+    }
+    Object.keys(action.data).forEach((key) => {
+        if (!nextState.items.has(key)) {
+            nextState.items.add(key);
+        }
+    });
+    return nextState;
+}
+
+export async function forceLogoutIfNecessary(err, dispatch) {
```

Comment:
Can you maybe move this to `actions/helpers.js`? Having Client calls in the utils feels too much like `utils.jsx` in the platform repo which is just full of random dependencies